### DISCUSSION TITLE
Elastix results interpretation help

### DISCUSSION THREAD

**Discussion by Misham2941** [SuperElastix/elastix#999] [Q&A]
Hi I am new to Elastix and image registration. I am currently doing a university project using Elastix as I find image registration very interesting.
In order to test Elastix and understand its behavior I was given 2 scans of a formal head where no deformation has occurred (the scans are identical).

From my limited experience and knowledge, I believe that Elastix will find the best optimal transformation to align the images which should be close to 0 pixels as the bone has undergone no deformation. Through researching, I found that rigid registration is sufficient enough for bone registration and so I used the developer recommended rigid registration parameter files. 

I performed registration on my scans and viewed the results using 3D slicer. However, as I am new to this process, I am having trouble interpreting the results.  Can someone please help me understand the results and how errors can be reduced during registration process? Furthermore, I want to know where I can go next to determine strain based on Elastix's results. I have attached some files for viewing I used 3D slicer. The first image is the deformation field, and the second image is the results file.  I have also attached the tranformparameters Elastix outputted and Rigid parameters file I used.

![Deformation field](https://github.com/SuperElastix/elastix/assets/145965419/46fea0e9-5dfb-4d73-948d-af9d6c5c8174)
![Results file](https://github.com/SuperElastix/elastix/assets/145965419/ca60f7da-bba3-4a4d-b76e-709cf64a10a8)
[TransformParameters.0.txt](https://github.com/SuperElastix/elastix/files/13532923/TransformParameters.0.txt)
[Parameters_Rigid.txt](https://github.com/SuperElastix/elastix/files/13532924/Parameters_Rigid.txt)

**Comment by raacampbell** [comment]
I would suggest you first overlay the fixed image and the results file as a red/green pair to see how well the align. Compare to the original fixed and moving image.
